pycgm/pycgm.py

```diff
@@ -0,0 +1,5263 @@
+        v1 = a - c
+        v2 = b - c
+
+        # v3 is np.cross vector of v1, v2
+        # and then it is normalized.
+        v3 = np.cross(v1, v2)
+        v3 = v3 / np.linalg.norm(v3)
+
+        m = (b + c) / 2.0
+        length = np.subtract(b, m)
+        length = np.linalg.norm(length)
+
+        theta = acos(delta / np.linalg.norm(v2))
+
+        cs = cos(theta * 2)
+        sn = sin(theta * 2)
+
+        ux = v3[0]
+        uy = v3[1]
+        uz = v3[2]
+
+        # This rotation matrix is called Rodrigues' rotation formula.
+        # In order to make a plane, at least 3 markers are required which means
+        # three physical markers on the segment can make a plane.
+        # Then the orthogonal vector of the plane will be rotating axis.
+        # Joint center is determined by rotating the one vector of plane around rotating axis.
+
+        rot = np.matrix([[cs + ux ** 2.0 * (1.0 - cs), ux * uy * (1.0 - cs) - uz * sn, ux * uz * (1.0 - cs) + uy * sn],
+                         [uy * ux * (1.0 - cs) + uz * sn, cs + uy ** 2.0 * (1.0 - cs), uy * uz * (1.0 - cs) - ux * sn],
+                         [uz * ux * (1.0 - cs) - uy * sn, uz * uy * (1.0 - cs) + ux * sn, cs + uz ** 2.0 * (1.0 - cs)]])
+        r = rot * (np.matrix(v2).transpose())
+        r = r * length / np.linalg.norm(r)
+
+        r = np.array([r[0, 0], r[1, 0], r[2, 0]])
+        mr = np.array([r[0] + m[0], r[1] + m[1], r[2] + m[2]])
+
+        return mr
+
+    @staticmethod
+    def wand_marker(rsho, lsho, thorax_axis):
+        """Wand Marker Calculation function
+
+        Takes in a dictionary of x,y,z positions and marker names.
+        and takes the thorax axis.
+        Calculates the wand marker for calculating the clavicle.
+
+        Markers used: RSHO, LSHO
+
+        Parameters
+        ----------
+        rsho, lsho : ndarray
+            A 1x3 ndarray of each respective marker containing the XYZ positions.
+        thorax_axis : ndarray
+            A 4x3 ndarray that contains the thorax origin and the
+            thorax x, y, and z axis components.
+
+        Returns
+        -------
+        wand : ndarray
+            Returns a 2x3 ndarray containing the right wand marker x, y, and z positions and the
+            left wand marker x, y, and z positions.
+
+        Examples
+        --------
+        >>> import numpy as np
+        >>> from .pycgm import CGM
+        >>> rsho, lsho = np.array([[428, 270, 1500],
+        ...                        [68, 269, 1510]])
+        >>> thorax_axis = np.array([[256, 364, 1459],
+        ...                         [256, 365, 1459],
+        ...                         [257, 364, 1459],
+        ...                         [256, 354, 1458]])
+        >>> CGM.wand_marker(rsho, lsho, thorax_axis)
+        array([[ 255.76812462,  364.        , 1459.9727455 ],
+               [ 256.26181402,  364.        , 1459.96511834]])
+        """
+
+        # REQUIRED MARKERS:
+        # RSHO
```

Comment:
this comment doesnt seem to match.  If the docstrings are correct, these comments can be reduced to "Get required markers" (but it seems like this is not the case here)